assets/js/project.js

```diff
@@ -83,11 +103,25 @@ Program(
 ProgramDescription(
   $projectDescriptionCredits.data('project-id'),
   $projectDescriptionCredits.data('trans-more-info'),
-  $projectDescriptionCredits.data('trans-less-info')
+  $projectDescriptionCredits.data('trans-less-info'),
+  $projectDescriptionCredits.data('trans-translation-saved'),
+  $projectDescriptionCredits.data('trans-translation-deleted'),
+  descriptionSelect,
+  closeEditorDialog,
+  keepOrDiscardDialog,
+  new SnackbarNotification('description'),
```

Comment:
Should be translatable string?

Reply:
Updated to a translatable string for description and credits sections